### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by nadjaheitmann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ for how to install Foreman plugins
 
 | Foreman Version | Katello Version | Plugin Version |
 | --------------- | --------------- | -------------- |
+| 2.3             | 3.18            | ~> 1.8.9       |
```
Comment (m-bucher) [inline README.md:16]: I guess, this is in preparation for the next version!?

**Review by m-bucher — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nadjaheitmann) [reply README.md:16]: Yes. Is it better to add this with the version bump?

File: lib/foreman_scc_manager/engine.rb [hunk 2]
```diff
@@ -22,8 +22,8 @@ class Engine < ::Rails::Engine
 
     initializer 'foreman_scc_manager.register_plugin', :before => :finisher_hook do |_app|
       Foreman::Plugin.register :foreman_scc_manager do
-        requires_foreman '>= 1.22'
-        requires_foreman_plugin 'katello', '>= 3.12.0'
+        requires_foreman '>= 2.1'
+        requires_foreman_plugin 'katello', '>= 3.16.0'
```
Comment (nadjaheitmann) [inline lib/foreman_scc_manager/engine.rb:26]: Is this okay or shall we also set the versions to 2.3/3.18 ?

File: README.md (hunk 1 above)
Comment (m-bucher) [reply README.md:16]: Maybe, but it is fine here as well.

File: lib/foreman_scc_manager/engine.rb (hunk 2 above)
Comment (m-bucher) [reply lib/foreman_scc_manager/engine.rb:26]: Only if it is really incompatible and does not work with <2.3/3.18

**Pull request merged**